src/components/screens/EventRecord.js

```diff
@@ -0,0 +1,147 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { SafeAreaView, View } from 'react-native';
+
+import { device, texts } from '../../config';
+import { HtmlView } from '../HtmlView';
+import { Image } from '../Image';
+import { Logo } from '../Logo';
+import { RegularText } from '../Text';
+import { Title, TitleContainer, TitleShadow } from '../Title';
+import { ScrollWrapper, Wrapper } from '../Wrapper';
+import { PriceCard } from './PriceCard';
+import { InfoCard } from './InfoCard';
+import { OperatingCompanyInfo } from './OperatingCompanyInfo';
+import { OpeningTimesCard } from './OpeningTimesCard';
+
+/* eslint-disable complexity */
+/* TODO: refactoring? */
+export const EventRecord = ({ data }) => {
+  const [page, setPage] = useState({});
+
+  useEffect(() => {
+    const {
+      addresses,
+      category,
+      contacts,
+      dataProvider,
+      description,
+      mediaContents,
+      prices,
+      title,
+      webUrls,
+      operatingCompany,
+      openingHours
+    } = data;
+
+    setPage({
```

Comment:
I will remove this `setPage...` in resolving merge conflict while merging to master. So we get rid of this extra step like already done in `PointOfInterest`.